backend/app/store/service/tree.go

```diff
@@ -145,3 +159,81 @@ func (t *Tree) sortNodes(sortType string) {
 		}
 	})
 }
+
+// limit limits number of comments in tree and sets countLeft and lastLimitedComment,
```

Comment:
I think there is no direct test for this non-trivial function, but rather only via MakeTree? It's up to you, but I will feel better if we have some.